src/tests/utils.ts

```diff
@@ -0,0 +1,27 @@
+import BN from 'bn.js'
+import { transfer } from 'near-api-js/lib/transaction'
+import { KeyPair } from 'near-api-js/lib/utils'
+import { parseNearAmount } from 'near-api-js/lib/utils/format'
+import { base_decode } from 'near-api-js/lib/utils/serialize'
+import { NearRpcProvider } from '../near-rpc-provider'
+
+export async function getSignedTransaction(provider: NearRpcProvider) {
+  const encodedKey = 'ed25519:2xrYyynoFs8RCwsfP1zoVctcMLYSiW9SwC82WJZTQdU3oV9bKjGtj7sD53temnixqKV6sZHikTBf3VxDvdkZfN3p'
```

Comment:
where is this key coming from?

Reply:
it's the key for the `blockcoders-test` account. We could provide it via the env, but since it's a testing account maybe it doesn't matter?